app/src/main/kotlin/org/fossify/contacts/activities/SettingsActivity.kt

```diff
@@ -165,6 +166,14 @@ class SettingsActivity : SimpleActivity() {
         }
     }
 
+    private fun setupEnableNumberFormatting(){
+        binding.enableFormattingPhoneNumber.isChecked = config.showPhoneNumbersFormatting
+        binding.settingsShowPhoneNumberFormattingHolder.setOnClickListener{
+            binding.enableFormattingPhoneNumber.toggle()
+            config.showPhoneNumbersFormatting = binding.enableFormattingPhoneNumber.isChecked
+        }
+    }
+
```

Comment:
I think "Format phone numbers" is a better name and fits nicely with the existing "Show phone numbers" preference. Please rename these symbols accordingly.